install/kimai-install.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: MickLesk
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+#add_core_dependencies
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  apt-transport-https \
+  sudo \
+  mc \
+  curl \
+  apache2 \
+  git \
+  expect \
+  composer \
+  mariadb-server \
+  libapache2-mod-php \
+  php8.2-{mbstring,gd,intl,pdo,mysql,tokenizer,zip,xml} 
+msg_ok "Installed Dependencies"
+
+msg_info "Setting up Database"
+DB_NAME=kimai_db
+DB_USER=kimai
+DB_PASS=$(openssl rand -base64 18 | tr -dc 'a-zA-Z0-9' | head -c13)
+MYSQL_VERSION=$(mysql --version | grep -oP 'Distrib \K[0-9]+\.[0-9]+\.[0-9]+')
+sudo mysql -u root -e "CREATE DATABASE $DB_NAME;"
```

Comment:
```suggestion
mysql -u root -e "CREATE DATABASE $DB_NAME;"
```
no need when root user